Project-iOS/XLProjectName/XLProjectName/Helpers/Extensions/AppDelegate+XLProjectName.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  AppDelegate+XLProjectName.swift
+//  XLProjectName
+//
+//  Created by XLAuthorName ( XLAuthorWebsite )
+//  Copyright © 2016 'XLOrganizationName'. All rights reserved.
+//
+
+import Foundation
+import Fabric
+import Alamofire
+import Eureka
+import Crashlytics
+
+extension AppDelegate {
+
+    func setupCrashlytics() {
+        Fabric.with([Crashlytics.self])
+        Fabric.sharedSDK().debug = Constants.Debug.crashlytics
+    }
+
+    // MARK: Alamofire notifications
+    func setupNetworking() {
+        NSNotificationCenter.defaultCenter().addObserver(
```

Comment:
I think it's not necessary to remove the observer since we want those notifications through the entire life cycle of the app. What do you think?